pkg/api/api_default_service.go

```diff
@@ -932,6 +994,34 @@ func (cfm *CfmApiService) HostsPost(ctx context.Context, credentials openapi.Cre
 	return openapi.Response(http.StatusCreated, h), nil
 }
 
+// HostsUpdateById -
+func (cfm *CfmApiService) HostsUpdateById(ctx context.Context, hostId string, newHostId string) (openapi.ImplResponse, error) {
+	// Make sure the hostId exists
+	// Get the host information from the manager level and is used for renaming
+	host, err := manager.GetHostById(ctx, hostId)
+	if err != nil {
+		return formatErrorResp(ctx, err.(*common.RequestError))
+	}
+
+	// Make sure the newHostId doesn't exist
+	_, exist := manager.GetHostById(ctx, newHostId)
```

Comment:
There's no need to call the same function twice (Appliance or Host).
See appliance comment.